styles/output/webview-generic.css

```diff
@@ -1,5 +1,5 @@
-@import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital@0;1&display=swap");
-@import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap");
+@import url("./PHILZZZ/IBM+Plex+Sans:ital@0;1.css");
+@import url("./PHILZZZ/IBM+Plex+Mono.css");
```

Comment:
Using local fonts for webview doesn't cause any problems, right?

Reply:
Hmm :thinking: Good question. When I'm adding `&content-style` to url of locally created version of webview it's working properly. Not sure how styles are injected in other processes. We are not using styles created in our `ce-styles` repo anywhere yet, right? So I'm not sure how I can check that :thinking:

Reply:
Also, this change was done within Phil's PR https://github.com/openstax/ce-styles/pull/289, just styles wasn't generated for webview, so maybe I will give it a try :thinking: What do you think?

Reply:
I'm thinking now, that since it's not related with this issue, I will merge it for now and make sure how it's should be tested on the channel. Maybe a separate card be needed for that :thinking:

Reply:
Nevermind :smile: Phil already made a fix and merge it to main instead of p.p https://github.com/openstax/ce-styles/commit/ca45767048271648966a69d5f0f9f8818b06ecc4. I will update p.p branch